charts/govuk-rails-app/templates/nginx-configmap.yaml

```diff
@@ -35,12 +65,22 @@ data:
         proxy_connect_timeout {{ .Values.nginxProxyConnectTimeout }};
         proxy_read_timeout    {{ .Values.nginxProxyReadTimeout }};
 
+        add_header Strict-Transport-Security $sts_default;
+        add_header X-Frame-Options DENY;
```

Comment:
As discussed, probably a good idea to omit this if it wasn't in the original config. The homepage seems to have `x-frame-options: ALLOWALL`, for example (presumably coming from the application itself?)